src/components/ElementBar/DuplicateActiveElementButton/DuplicateActiveElementButton.test.tsx

```diff
@@ -30,7 +32,9 @@ describe('duplicate', () => {
       strokeColor: '#ff0000',
     };
 
-    expect(duplicate(element, 40)).toEqual({
+    expect(
+      duplicate(element, 40, calculateBoundingRectForElements([element])),
```

Comment:
Would be good to have a test that takes several elements to calculate bounding rect and checks that elements are within whiteboard when boundaries could be exceeded.

Reply:
Okay